Exercise : 6.1 Load External Content

imvishi opened the pull request.

Commit:
Exercise : 6.1 Load External Content

Ankita9818 commented on exercises/js/blogloader.js:
```diff
@@ -0,0 +1,41 @@
+    this.blogs.each(function(i, header) {
+      const div = $('<div ></div>')
+      div.insertAfter($(header));
+      $(header).data('target', div);
```
$(header) is called multiple times. Please optimize

imvishi replied:
Tried to optimize it.

Ankita9818 commented on exercises/js/blogloader.js:
```diff
@@ -0,0 +1,41 @@
+    this.blogs.each(function(i, header) {
+      const div = $('<div ></div>')
+      div.insertAfter($(header));
+      $(header).data('target', div);
```
$(header) is called multiple times. Please optimize

imvishi replied:
Tried to optimize it.

Ankita9818 commented on exercises/js/blogloader.js:
```diff
@@ -0,0 +1,41 @@
+      $(this).click(function(event) {
+        event.preventDefault();
+        const header = $(this)
+        const postId = $(header).data('postId');
```
No need of $(header)

imvishi replied:
Done.

Ankita9818 reviewed (changes requested):
Avoid force pushing as one might need the commit history for reviewing changes.

Ankita9818 commented on exercises/js/blogloader.js:
```diff
@@ -0,0 +1,43 @@
+
+  bindEvents() {
+    this.blogs.each(function() {
+      $(this).click(function(event) {
```
$(this) is used twice so can be kept out in a variable.

imvishi replied:
Fixed.

Ankita9818 commented on exercises/js/blogloader.js:
```diff
@@ -0,0 +1,43 @@
+    this.blogs = $(this.blogLoaderObject.blogHeaderSelector);
+    this.blogs.each(function(i, header) {
+      const headerElement = $(header);
+      const div = $('<div ></div>')
```
Is the space in div creation is intentional?? Why??

imvishi replied:
This space is not intentional. Removed.

Ankita9818 reviewed (changes requested):
As mentioned in previous review , Please do not force push.

Commit:
Fixed following issues:

- Removed unnecessary space from div tag.
- used variable for storing multiple used $(this).
